samples/net/lwm2m_client/prj.conf

```diff
@@ -1,3 +1,6 @@
+# MAIN
+CONFIG_MAIN_STACK_SIZE=2048
```

Comment:
Since it's only needed for OT, I think it could be placed in the `overlay-ot.conf` not to increase the memory footprint for other platforms.